Preview.js

```diff
@@ -0,0 +1,218 @@
+import React from 'react'
```

Comment:
This file seem duplicated and not in use

Reply:
Will fix.